mod/search.php

```diff
@@ -214,7 +214,7 @@ function search_content(App $a) {
 			FROM `item` %s
 			WHERE %s AND (`item`.`uid` = 0 OR (`item`.`uid` = %s AND NOT `item`.`global`))
 				$sql_extra
-			GROUP BY `item`.`uri` ORDER BY `item`.`id` DESC LIMIT %d , %d",
+			GROUP BY `item`.`uri`, `item`.`id` ORDER BY `item`.`id` DESC LIMIT %d , %d",
```

Comment:
@Hypolite Could you please have a look again at this one? The additional GROUP BY seems to give fine results on my instance.